docs/notebooks/translation_matrix.ipynb

```diff
@@ -0,0 +1,528 @@
0IoSKE8KsQQvcs\n9ytJaqOqqipmzfoV8GdgObCY997bwIgRIzj66KP53ve+t9MIVuP5XDtMnjyZDRs2cMQRRzBt2jSO\nOeaY+vt+9rOfceqpp3L88cdz4IEH1q+fOXMmS5Ys4aijjuLII4/kpptu2k1HKeW30Nx5+voGITwG\n9Gm4CojAFcAtMcZPN2j7Toxx30aPH0LmXf6lGOOSEMIvgXUxxh83s7/44x9/dFdpaSmlpaWf6KAk\n5a/y8nLKy8vrb0+fPp0YY9P/wxeYEELcVZ+aD+bMmcM55/yUTPja4YvccssPmTRpUpu3O2bMGGbM\nmEFJSUnWNUqFIoTQpj5slwFsFzutAkpjjG+FEPYHFsQY+zdq0wd4Nsb4ubrbxwOXxRi/3Mw2C6ID\nk5Qbbe288lGh9F9VVVUMGLDjb+NBQCUwgpdffp7+/fu3/OAWnHDCCfziF78wgKlDaWsflu0pyHnA\nOXXLk4A/NG5Qd4pyTQjhi3WrTgT8zLEkpaR///5MmXIeMAL4IjCCKVPOyyp8ATzxxBOGL6mVsh0B\n+zTwO+CzwGoyl6F4L4RwAHBzjPG0unZHkbkMRRdgFfDtpi5XUde2IP6ClJQbjoClp6qqiueee45h\nw4ZlHb6kjiqVU5C7Q6F1YJKyYwCTVMjSOgUpSZKkT8gAJkmSlDADmCRJUsIMYJIkSQkzgEmSJCXM\nACZJkpQwA5gkSVLCDGCSJEkJM4BJkiQlzAAmSZKUMAOYJElSwgxg7cT111/PgAED2Hffffmv//qv\ntMuRJEkt8Mu424n+/fvz+OOPc+CBB6ZdivSJ+GXckgqZX8bdgU2ePJlVq1Yxbtw4fvnLX3LRRRex\nfv16iouL69t8+OGHHHLIIdTW1ta3HTp0KKNHj2bFihXpFS9JUgdkAGsHbrzxRg466CDKy8vp1asX\nIQT22msvjj76aP70pz8B8OCDD3LKKafQuXNnzj//fGbNmsXixYu5+uqrmTx5cspHIElSx1KUdgHK\nTk1NDdXV1dTW1tL41Mc3vvEN7rrrLkaPHs2dd97Jd7/7XT744AMWLlzIhAkT6ttv3bo1jdIlSeqw\nDGAF7I477uLccy+ka9di1q9/nXvvvY+uXT96Sb/yla/wwx/+kLVr11JRUcEJJ5zAhg0b6NWrFxUV\nFSlWLklSx+YpyAJVU1PDuedeyMaNC1i37nli3J+LL/4+69evr2/To0cPjjnmGC655BJOO+00Qgj0\n7NmTQw89lHvuuae+XWVlZRqHIElSh2UAK1DV1dV07VoMDKpb040uXT7L22+/vVO7M888k7lz53LW\nWWfVr5s7dy5lZWUMHjyYI488knnz5iVWtyRJ8jIUBaumpoa+ffuxceMCMiGsku7dx7B69TJ69+6d\ndnlSq3kZCkmFzMtQdDC9e/emrGw23buPYa+9SujefQxlZbMNX5IkFQBHwArcjk9BFhcXG75UkBwB\nk1TI2tqHGcAkpcoAJqmQeQpSkiSpQBjAJEmSEmYAkyRJSpgBTJIkKWEGMEmSpIQZwCRJkhJmAJMk\nSUqYAUySJClhBjBJkqSEGcAkSZISZgCTJElKmAFMkiQpYQYwSZKkhBnAJEmSEmYAkyRJSpgBTJIk\nKWEGMEmSpIQZwCRJkhJmAJMkSUqYAUySJClhBjBJkqSEGcAkSZISZgCTJElKmAFMkiQpYVkFsBBC\nrxDCoyGE5SGER0IIezfTbmoI4a8hhMoQwtwQQtds9lsIysvL0y4hZzyW/NNejiNtIYSvhxBeCiHU\nhhBKWmh3SghhWQhhRQjhsiRrzEa+/Z7kWz2QfzVZT8vyrZ5sZDsCdjkwP8Z4OPAEMLVxgxBCX+A8\n4OgY4yCgCDgry/3mvfb0S+Kx5J/2chx54EXgDOBPzTUIIXQCZgFjgSOAiSGEfsmUl518+z3Jt3og\n/2qynpblWz3ZyDaAnQ7MqVueA3y1iTbrgS1AjxBCEbAn8EaW+5WkrMUYl8cYVwKhhWbDgJUxxtUx\nxq3AnWT6Pklqs2wD2H4xxrcAYoxvAvs1bhBjXAvMAF4D/g68F2Ocn+V+JSkpBwFrGtx+vW6dJLVZ\niDG23CCEx4A+DVcBEbgCuCXG+OkGbd+JMe7b6PGfAx4EjgfWAfcAd8cYb29mfy0XJKndiTG2NAKV\nlRb6sB/GGB+oa7MA+F6MsaKJx38NGBtjPL/u9j8Dw2KMFzfR1v5L6oDa0ocVtWKjJzV3XwjhrRBC\nnxjjWyGE/YH/aaLZMcAzMcZ36x7ze+BYoMkAtjs7YkkdT0t9WCv9HTikwe2D69Y1tS/7L0mtku0p\nyHnAOXXLk4A/NNFmOTAihLBHCCEAJwJVWe5XknKtufC0GPh8CKFv3Se4zyLT90lSm2UbwH4OnBRC\nWE4mWF0FEEI4IITwIECMcSlwK/A8sJRMJ/erLPcrSVkLIXw1hLAGGAE8GEJ4uG59wz6sFpgCPAr8\nFbgzxugfkZKysss5YJIkScqtVK+E354ugvgJLkpbHUJYGkL4SwjhuaTrbElrnucQwnUhhJUhhBdC\nCIOTrrE1dnUcIYTRIYT3QggVdT9XpFFna4QQyurmWla20KYQXpMWj6OQXpOG8q0Py5d+KN/6knzr\nE/LtfZ1v788QwsEhhCdC5gLuL4YQPvaBl7p2iTxHramnTc9RjDG1H+Bw4AtkLuJa0kybTsArQF+g\nC/AC0C/Nupup8+fAD+qWLwOuaqbdKqBX2vW25XkGxgF/rFseDvw57brbeByjgXlp19rK4zkeGAxU\nNnN/3r8mrTyOgnlNGtWdV31YPvRD+daX5GOfkG/v63x7fwL7A4Prlj9FZi55mr9DrannEz9HqY6A\nxfZ1EcTWXJQWMseaj9/B2Zrn+XQy8/mIMS4C9g4h9CG/tPb3pSA+rRZjfBpY20KTQnhNWnMcUCCv\nSUN52IflQz+Ub31J3vUJ
```

Comment:
Please add non-trivial examples (fruits, animals, random words, etc). Also please add your experiments in the distinct notebook.